site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: doc
+title: Generic Messenger
+description: Adding and Configuring Messengers - Generic
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Set Up a Generic Messenger
+
+Currently the only supported transport for a messenger is SMS. If you need to connect to an SMS provider other than Twilio, you can use the Generic messenger configuration.
+
+The generic messenger allows for a simple JSON REST API integration which allows you to receive a message from FusionAuth and then make an API call to any third party SMS provider you wish.
+
+image::messengers/add-messenger.png[Messenger Home Screen, width=1200]
+
+To create a new generic messenger, navigate to [breadcrumb]#Settings -> Messengers#. +
+Simply click `Add Generic Messenger`
+
+=== Add Generic Messenger
+
+image::messengers/add-generic-messenger-home-crop.png[Add Generic Messenger Home, role=bottom-cropped, width=1200]
+
+Just complete the following fields:
```

Comment:
Can you please remove `just`? I need to get out of the habit of doing this too, but such wording can intimidate folks to whom this isn't a 'simple' task :) .